BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -33,6 +33,30 @@ extension StackElement {
         configure(&self)
     }
 
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - elementsBuilder: A block containing all elements to be included in the stack.
+    /// - Note: If element is a StackChild, then traits and key will be pulled from the element, otherwise defaults are passed through.
+    public init(@ElementBuilder<StackChild> elementsBuilder: () -> [StackChild]) {
+        self.init()
+        children = elementsBuilder().map { stackChild in
+            (stackChild.element, stackChild.traits, stackChild.key)
+        }
+    }
+
+    /// Initializes stack transforming an array of some item into an array of elements.
+    /// - Parameters:
+    ///   - items: The array of items to transform.
+    ///   - transform: Closure to be applied to each item `T` which transforms that item into an `Element`.
+    ///   - configure: A closure used to modify the stack.
+    public init<T>(_ items: [T], transform: (T) -> StackChild, configure: (inout Self) -> Void = { _ in }) {
```

Comment:
Minor nit: Swift has moved away from `T`, `U`, etc, to names that reflect the actual value of the generic (eg `Element`); might be nice to do that here too.

Reply:
I actually removed this init since I added the ability for the builder to take in `[Element]` and `[Child]` and its just as easy to call `items.map(...)` inside the block.